include/chopper/count/configuration.hpp

```diff
@@ -8,16 +8,17 @@ namespace chopper::count
 
 struct configuration
 {
-    std::filesystem::path data_file{};
-    std::filesystem::path output_filename{};
-    std::filesystem::path hll_dir{};
+    std::filesystem::path data_file;
+    std::string output_prefix{}; // given by the user
+    std::filesystem::path weights_filename{};   // set internally
```

Comment:
Not sure if weights is the right name. But `count` isn't right anymore since we only use sketches now.

Reply:
Mhm, aren't they still counts? Now we just count sketches instead of minimisers.

Reply:
mhm, they are count estimates :shrug: ?